### ISSUE TITLE
Exception when trying to find usages of a Spring bean

### ISSUE THREAD

**Issue by MarioAriasC** [ishchenko/idea-mini-ibatis#9]
This exception arise when I try to see the usages (ALT + F7) of a Spring Bean 

```
Error during dispatching of java.awt.event.InvocationEvent[INVOCATION_DEFAULT,runnable=com.intellij.usages.impl.UsageViewImpl$4@34a771d8,notifier=null,catchExceptions=false,when=1375351296386] on apple.awt.CToolkit@665c4d90
java.lang.AssertionError
    at net.ishchenko.idea.minibatis.IdentifiableStatementHierarchyProvider.getTarget(IdentifiableStatementHierarchyProvider.java:46)
    at com.intellij.usageView.impl.UsageContextCallHierarchyPanel$Provider.isAvailableFor(UsageContextCallHierarchyPanel.java:67)
    at com.intellij.usages.impl.UsageViewImpl$8.value(UsageViewImpl.java:291)
    at com.intellij.usages.impl.UsageViewImpl$8.value(UsageViewImpl.java:288)
    at com.intellij.util.containers.ContainerUtil.findAll(ContainerUtil.java:775)
    at com.intellij.util.containers.ContainerUtil.filter(ContainerUtil.java:752)
    at com.intellij.usages.impl.UsageViewImpl.setupCentralPanel(UsageViewImpl.java:288)
    at com.intellij.usages.impl.UsageViewImpl.access$1100(UsageViewImpl.java:86)
    at com.intellij.usages.impl.UsageViewImpl$4.run(UsageViewImpl.java:211)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
    at java.awt.EventQueue.access$400(EventQueue.java:82)
    at java.awt.EventQueue$2.run(EventQueue.java:676)
    at java.awt.EventQueue$2.run(EventQueue.java:674)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
    at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:700)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:525)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:348)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
```

**Comment by ishchenko** [comment]
`HirarchyProvider#getTarget` gets `null` when trying to get `Editor` instance from `dataContext`

**Comment by ishchenko** [comment]
Fixed in build 1.13, uploaded to JetBrains plugins repo. Thanks for you report, MarioAriasC!
